forem/forem
# Theming changes part 1: Rename default and night_theme

## What type of PR is this? (check all applicable)


## Description

This PR renames two existing themes:

1. Default -> Light Theme
2. Night Theme -> Dark Theme

## Related Tickets & Documents

#14835 

## QA Instructions, Screenshots, Recordings

* Change your theme via http://localhost:3000/settings/customization
* Keep in mind that you theme changes often require a reload
* Both the light and dark theme should work as expected

### UI accessibility concerns?

None, this is just a rename for now.

## Added/updated tests?


## [Forem core team only] How will this change be communicated?
